lld/ELF/Writer.cpp

```diff
@@ -902,11 +902,12 @@ enum RankFlags {
   RF_NOT_ALLOC = 1 << 26,
```

Comment:
This ranking mechanism is a bit difficult to understand, but changing it is out of scope of this change.